### PR TITLE
docs(scheduler):resource grouping documentation added

### PR DESCRIPTION
**Pull request by xristianstefanov**
- Added Grouping in the Scheduler documentation
- New sections for Resource Grouping in all the available views

### CODE REVIEW

File: components/scheduler/views/day.md [hunk 1]
```diff
@@ -99,6 +100,17 @@ In this article:
...
+
+You can configure the Day view to display events that are [grouped]({%slug scheduler-grouping%}) by a resource.
+
+>caption The result from the code snippet below.
+
+![](images/scheduler-resource-grouping-day-view.png)
+
+>caption Resource Grouping in a Day view.
+
+@[template](/_contentTemplates/scheduler/views.md#code-snippet)
```
Comment (marin-bratanov) [inline components/scheduler/views/day.md:113]: Use a more descriptive heading for the template

File: components/scheduler/grouping.md [hunk 2]
```diff
@@ -0,0 +1,573 @@
+---
+title: Grouping
```
Comment (marin-bratanov) [inline components/scheduler/grouping.md:2]: I recommend you add to the url and various title the keyword "resource grouping"

**Comment by marin-bratanov** [comment]
I recommend you rename the `#code-snippet` name in `(/_contentTemplates/scheduler/views.md#code-snippet)` to make it more meaningful. Will try to build and review in more detail over the weekend.

**Review by marin-bratanov — CHANGES_REQUESTED** [review]
The OnInitializedAsync in the component should not have to generate resources, the service should provide them. Make sure that code is cleaned up.

Rename the headings for resource grouping in the inidividual view articles to include the view name (e.g. "Resource Grouping in the Day View") as that's the first examples people will see when landing on the Resource Grouping page.

In the Resource Grouping article, consider moving the links to the four views examples in the basics section - so people have seen one example and are familiar with the new tag and its settings before looking at rather complex code.

Add a screenshot for grouping by multiple resources. In fact, consider making headings for them, it is a nice thing to point out to people. You can have three sections - Basics, Single resource grouping (that also links to individual views examples) and Multipel Resource Grouping.

File: components/scheduler/views/day.md [hunk 3]
```diff
@@ -99,6 +100,17 @@ In this article:
 
 @[template](/_contentTemplates/scheduler/views.md#day-slots-explanation)
 
+## Resource Grouping in the Day View
+
+You can configure the Day view to display events that are [grouped]({%slug scheduler-resource-grouping%}) by a resource.
```
Comment (dimodi) [inline components/scheduler/views/day.md:105]: better SEO with more keywords in the link:

```suggestion
You can configure the Day view to display events that are [grouped by a resource]({%slug scheduler-resource-grouping%}).
```

File: components/scheduler/views/month.md [hunk 4]
```diff
@@ -95,9 +100,17 @@ A day shows up to two events and if there are more - an ellipsis button provides
     }
 }
 ````
+## Resource Grouping in the Month View
+
+You can configure the Month view to display events that are [grouped]({%slug scheduler-resource-grouping%}) by a resource.
```
Comment (dimodi) [inline components/scheduler/views/month.md:105]:
```suggestion
You can configure the Month view to display events that are [grouped by a resource]({%slug scheduler-resource-grouping%}).
```

File: components/scheduler/views/multiday.md [hunk 5]
```diff
@@ -102,6 +103,17 @@ In this article:
 
 @[template](/_contentTemplates/scheduler/views.md#day-slots-explanation)
 
+## Resource Grouping in the MultiDay View
+
+You can configure the MultiDay view to display events that are [grouped]({%slug scheduler-resource-grouping%}) by a resource.
```
Comment (dimodi) [inline components/scheduler/views/multiday.md:108]:
```suggestion
You can configure the MultiDay view to display events that are [grouped by a resource]({%slug scheduler-resource-grouping%}).
```

File: components/scheduler/views/week.md [hunk 6]
```diff
@@ -100,6 +101,17 @@ In this article:
 
 @[template](/_contentTemplates/scheduler/views.md#day-slots-explanation)
 
+## Resource Grouping in the Week View
+
+You can configure the MultiDay view to display events that are [grouped]({%slug scheduler-resource-grouping%}) by a resource.
```
Comment (dimodi) [inline components/scheduler/views/week.md:106]:
```suggestion
You can configure the MultiDay view to display events that are [grouped by a resource]({%slug scheduler-resource-grouping%}).
```

File: components/scheduler/resource-grouping.md [hunk 7]
```diff
@@ -0,0 +1,559 @@
+---
+title: Resource Grouping
+page_title: Scheduler - Resource Grouping
+description: Resource Grouping in the Scheduler for Blazor.
```
Comment (dimodi) [inline components/scheduler/resource-grouping.md:4]: Use different keywords or different grammar forms:

```suggestion
description: Group Resources in the Scheduler for Blazor.
```

File: components/scheduler/resource-grouping.md [hunk 8]
```diff
@@ -0,0 +1,559 @@
...
+slug: scheduler-resource-grouping
+tags: telerik,blazor,scheduler,resource,grouping
+published: true
+position: 33
+---
+
+# Scheduler Resource Grouping
+
+The feature allows grouping Scheduler views by one or more resources. All available supported views: `Day`, `Week`, `MultiDay` and `Month` can render both `horizontal` and `vertical` grouping.
```
Comment (dimodi) [inline components/scheduler/resource-grouping.md:13]: We group events, not views, right?

```suggestion
The feature allows grouping Scheduler events by one or more resources. All available supported views: `Day`, `Week`, `MultiDay` and `Month` can render both `horizontal` and `vertical` grouping.
```

File: components/scheduler/resource-grouping.md [hunk 9]
```diff
@@ -0,0 +1,559 @@
...
+
+## Basics
+
+In the grouping rendering in both horizontal and vertical orientation, the view tables are rendered next to each other. 
+
+Moving an appointment between the resource tables is allowed, and upon dropping, the appointment resource changes alongside the start date.
+
+### Define Settings
+
+To configure the group rendering, define a dedicated settings tag, called a `SchedulerGroupSettings` inside the `SchedulerSettings` tag.
```
Comment (dimodi) [inline components/scheduler/resource-grouping.md:31]: Removing self-explanatory details.

```suggestion
To configure the group rendering, define the `SchedulerGroupSettings` tag inside the `SchedulerSettings` tag.
```

File: components/scheduler/resource-grouping.md [hunk 10]
```diff
@@ -0,0 +1,559 @@
...
+
+The settings tag will have the following Parameters:
+
+* `Resources(List<string>)` - provides a list of one or more resource names, which will be used to group the scheduler by.
```
Comment (dimodi) [inline components/scheduler/resource-grouping.md:35]:
```suggestion
* `Resources(List<string>)` - provides a list of one or more resource names, which will be used to group the events.
```

File: components/scheduler/resource-grouping.md [hunk 11]
```diff
@@ -0,0 +1,559 @@
...
+* `Orientation(SchedulerGroupOrientation)` - has two values: `Horizontal(default)` and `Vertical`. Determines the direction in which the resource tables are rendered.
```
Comment (dimodi) [inline components/scheduler/resource-grouping.md:36]:
```suggestion
* `Orientation(SchedulerGroupOrientation)` - has two values: `Horizontal` (default) and `Vertical`. Determines the direction in which the resource tables are rendered.
```

**Review by dimodi — APPROVED** [review]

**Review by marin-bratanov — APPROVED** [review]
approving, I have not had time to check

**Pull request merged**